prow/plugins/trigger/pr.go

```diff
@@ -55,21 +56,21 @@ func handlePR(c client, trigger *plugins.Trigger, pr github.PullRequestEvent) er
 		}
 	case github.PullRequestActionReopened:
 		// When a PR is reopened, check that the user is in the org or that an org
-		// member had said "/ok-to-test" before building.
-		comments, err := c.GitHubClient.ListIssueComments(org, repo, num)
+		// member had said "/ok-to-test" before building, resulting in label ok-to-test.
+		labels, err := c.GitHubClient.GetIssueLabels(org, repo, num)
 		if err != nil {
 			return err
 		}
-		trusted, err := trustedPullRequest(c.GitHubClient, trigger, author, org, repo, comments)
+		trusted, err := trustedPullRequest(c.GitHubClient, trigger, author, org, repo, labels)
 		if err != nil {
 			return fmt.Errorf("could not validate PR: %s", err)
 		} else if trusted {
-			err = clearStaleComments(c.GitHubClient, pr.PullRequest, comments)
```

Comment:
Are we no longer adding the comment at the start of an untrusted PR?

Reply:
You're right, we do... I was hoping to save some API tokens by not listing all comments, but then how to remove that `waitingComment`?

Reply:
I don't think we can save the tokens, unfortunately

Reply:
And how about putting a more generic comment that works for both `needs-ok-to-test` and `ok-to-test` like:
> You are not a trusted member of the organization.
> If your PR still has the `needs-ok-to-test` label, it is waiting for a member to check that this patch is reasonable to test...
> If your PR has the `ok-to-test` label, tests will be run on commit or with the command /retest...

Then this comment could stay and we no longer have to list them for eventual pruning.
What do you think?

Reply:
That seems like a reasonable workaround but I wonder what @cjwagner and @kubernetes/federation-reviewers think

Reply:
That seems fine to me, although it is unclear if we need a comment at all if the PR has the `ok-to-test` label.

Reply:
Indeed, but we still need the initial comment explaining why tests are not run for non-members... that's why we should make it meaningful for both labels.